appConfigurations.js

```diff
@@ -20,7 +20,8 @@ const filteredWhitelist = fileTypeWhitelist.filter(item => item.indexOf('/') ===
 const truthies = ['true', 'True', 'TRUE', '1', 'yes', 'Yes', 'YES', 'y', 'Y'];
 const falsies = ['false', 'False', 'FALSE', '0', 'no', 'No', 'NO', 'n', 'N'];
 const isDev = () => process.env.NODE_ENV === 'development';
-
+const webChatClientBaseUrl = process.env.NODE_ENV === 'production' ? ('webchat-client.ctsc.hmcts.net') : ('webchat-client.pp.ctsc.hmcts.net');
```

Comment:
why we are hardcoding the url in code

Reply:
Before, directly webchat-client.ctsc.hmcts.net was used in the script. I am now setting prod and pre-prod value based on environment. It is the chat window domain URL

Reply:
I guess what @srochani is suggesting that we could have this set in a config/be determined by cnp-flux, which should make it easier for us to change in the future?

Reply:
Modified and default is prod